compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParserErrorHandler.java

```diff
@@ -1353,10 +1359,10 @@ private ParserRuleContext getNextRule(ParserRuleContext currentCtx, int nextLook
                     endContext(); // end parameter-list
                 }
                 if (parentCtx == ParserRuleContext.NIL_TYPE_DESCRIPTOR) {
-                    endContext();
-                    // After parsing nil type descriptor all the other parsing is same as next rule of simple type
-                    return getNextRuleForTypeDescriptor();
-                }
+                endContext();
+                // After parsing nil type descriptor all the other parsing is same as next rule of simple type
+                return getNextRuleForTypeDescriptor();
+            }
```

Comment:
Incorrect indentation. 
I'l merge this PR anyway. Please fix it in the next PR.

Reply:
sure